bin/kubectl-crossplane-stack-init

```diff
@@ -110,6 +119,13 @@ EOF
   echo 'Created config/stack/overrides/install.yaml' >&2
 }
 
+function check_for_existing_stack {
+  if [ -f "${INIT_DIR}/stack.env" -a -z "${FORCE_INIT}" ]; then
+    echo "${INIT_DIR}/stack.env already exists.  Use -f to force init." >&2
```

Comment:
Could you add a prefix like `"ERROR: ${INIT_DIR} . . . ` so it's more obvious when scanning that the init failed?

Reply:
How about following this example ([see my other comment for context](https://github.com/crossplaneio/crossplane-cli/pull/6#discussion_r321741824)).
```
$ kubectl foo  --help
Error: unknown command "foo" for "kubectl"
```

Reply:
I like these ideas! How about if we say we want this to be the style we're going for, and as we edit lines, we update them to match the style?